src/main/flight/pid.c

```diff
@@ -198,10 +198,10 @@ void resetPidProfile(pidProfile_t *pidProfile)
         .transient_throttle_limit = 0,
         .profileName = { 0 },
         .idle_min_rpm = 0,
-        .idle_adjustment_speed = 50,
-        .idle_p = 50,
-        .idle_pid_limit = 200,
-        .idle_max_increase = 150,
+        .dyn_idle_p_gain = 50,
```

Comment:
You are changing what individual parameters stands for, and what defaults then need - this means that the parameter group version will have to be incremented, or else unsuspecting users updating the firmware will get invalid settings without ever having touched them.

Reply:
Good point, will do, thanks.

Reply:
PG value incremented to 2